config/webpack.prod.js

```diff
@@ -216,17 +217,6 @@ module.exports = function (env) {
       new CriticalCssPlugin({
         src: 'index.html'
       }),
-
-      new CopyWebpackPlugin([
-       {
-         from: 'src/assets',
-         to: 'assets',
-       },
-       {
-         from: 'config/CNAME'
```

Comment:
We need the CNAME file to be copied over for Production purposes. Do you plan on adding this functionality back in?